integration/resource_lacework_vulnerability_exception_container_test.go

```diff
@@ -66,3 +65,44 @@ func TestVulnerabilityExceptionContainerCreate(t *testing.T) {
 	assert.Equal(t, "[CVE-2016-9840 CVE-2018-6940]", actualCves)
 	assert.Equal(t, "[map[name:adm-zip version:0.4.7] map[name:myUpdatedPackage version:1.1.0]]", actualPackages)
 }
+
+// TestVulnerabilityExceptionContainerCreateFixableUnset applies integration terraform:
+// => '../examples/resource_lacework_vulnerability_exception_container/fixable_unset'
+//
+// It uses the go-sdk to verify the created vulnerability exception,
+// applies an update and destroys it
+func TestVulnerabilityExceptionContainerCreateFixableUnset(t *testing.T) {
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "../examples/resource_lacework_vulnerability_exception_container/fixable_unset",
+		EnvVars:      tokenEnvVar,
+		Vars: map[string]interface{}{
+			"name":            "Terraform Vulnerability Exception Container Test",
+			"description":     "Vulnerability Exception Container created by Terraform",
+			"cves":            []string{"CVE-2016-9840", "CVE-2018-14599", "CVE-2018-6942"},
+			"package_name":    "myPackage",
+			"package_version": "1.0.0",
+		},
+	})
+	defer terraform.Destroy(t, terraformOptions)
+
+	// Create new Vulnerability Exception
+	create := terraform.InitAndApplyAndIdempotent(t, terraformOptions)
+
+	// check resource with fixable unset
+	unsetFixableResource := GetVulnerabilityExceptionProps(create, 1)
+	assert.Nil(t, unsetFixableResource.Data.VulnerabilityCriteria.FixableEnabled())
+
+	// Update Vulnerability Exception
+	terraformOptions.Vars = map[string]interface{}{
+		"name":            "Terraform Vulnerability Exception Container Test",
+		"description":     "Updated Vulnerability Exception created by Terraform",
+		"cves":            []string{"CVE-2016-9840", "CVE-2018-6940"},
+		"package_name":    "myUpdatedPackage",
+		"package_version": "1.1.0",
+	}
+
+	update := terraform.ApplyAndIdempotent(t, terraformOptions)
+	updateProps := GetVulnerabilityExceptionProps(update, 1)
+
+	assert.Nil(t, updateProps.Data.VulnerabilityCriteria.FixableEnabled())
```

Comment:
We need to add two tests (which means two Terraform applies)

- One is to SET the fixable field to either `true` or `false` (not nil)
- The second one to revert the change and UNSET the fixable field (put it back to nil)